src/rdkafka_admin.c

```diff
@@ -5210,6 +5210,7 @@ void rd_kafka_AlterConsumerGroupOffsets(
             rd_kafka_AlterConsumerGroupOffsetsResponse_parse,
         };
         rd_kafka_op_t *rko;
+        rd_kafka_topic_partition_list_t *copied_offsets;
```

Comment:
Why are we creating a new copy? Do we really need to preserve the order?

Reply:
I thinks it's better to avoid having side effects for an operation that is read only, also done in other places